tests/ui/flatten_and_doc.stderr

```diff
@@ -1,7 +1,5 @@
-error: proc-macro derive panicked
+error: methods and doc comments are not allowed for flattened entry
```

Comment:
Can't we point to the flatten or doc comment??

Reply:
Done, now we point to the flatten